docs/design/dialog-boxes.md

```diff
@@ -27,7 +27,6 @@ For a sample that implements a dialog box, see [Office Add-in Dialog API Example
 
 ## See also
 
-- [GitHub Development Resources](https://github.com/OfficeDev/Office-Add-in-UX-Design-Patterns-Code)
```

Comment:
Should we be replacing these with links to the "UX design patterns for Office Add-ins" article?